src/components/editor/PropertyResourceTemplate.jsx

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import React, { Component } from 'react'
+import PropertyTemplateOutline from './PropertyTemplateOutline'
+import PropTypes from 'prop-types'
+
+class PropertyResourceTemplate extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      collapse: false
+    }
+  }
+
+  isCollapsed  = () => {
```

Comment:
Do we need this empty function and it's call?

Reply:
Removed in latest commit